sdks/java/io/csv/build.gradle

```diff
@@ -31,4 +31,5 @@ dependencies {
     testImplementation project(path: ":sdks:java:core", configuration: "shadowTest")
     testImplementation library.java.junit
     testRuntimeOnly project(path: ":runners:direct-java", configuration: "shadow")
+    testImplementation project(path: ":sdks:java:io:common", configuration: "testRuntimeMigration")
```

Comment:
Notice this "testRuntimeMigration" is actually a workaround migrating to Gradle 7 (#16307) but somehow the TODO gets removed in #16430. 

CC: @lukecwik Is this configuration still a pending cleanup and are new projects able to get rid of this flag? For this PR we can leave it as is for now.

Reply:
@Abacn Thank you so much for reviewing.  FYI `testImplementation project(path: ":sdks:java:io:common")` causes: `error: package org.apache.beam.sdk.io.common does not exist`

Reply:
@Abacn per our conversation resolving comment.